src/index.js

```diff
@@ -60,22 +60,17 @@ export default function ({ types: t }) {
         }
       },
       MemberExpression(path, { opts = {} }) {
-
         let { node } = path;
 
-        opts.optimize = (opts.optimize === true) || false;
```

Comment:
The idea was to only allow boolean values for boolean options. But maybe truthy/falsy values are good enough. But the re-init really doesn't make sense.